### PR TITLE
📚  Add Nodelinter doc

### PR DESCRIPTION
**Pull request by gpene**

### CODE REVIEW

File: docs/nodes/creating-nodes/nodelinter.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      "enabled": false
+    },
+    "NAME_WITH_NO_CAMELCASE": {
+      "enabled": false
+    }
+  }
+}
+```
+
+Name this file `nodelinter.config.json` and place it anywhere in your local Nodelinter directory and it will be automatically detected. Alternatively, you can specify the custom config file and location using the `--config` option.
```
Comment (ivov) [inline docs/nodes/creating-nodes/nodelinter.md:63]: > in your local Nodelinter directory

When installing as a dep there is no such dir, config goes in nodes-base

**Review by ivov — CHANGES_REQUESTED** [review]
The whole point of the linter is to save us work, so somewhere should say that a clean lint is a requirement for submitting a PR.

File: docs/nodes/creating-nodes/nodelinter.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+```sh
+npm run nodelinter -- --<options>
+```
+
+## Options
+
+| Option            | Description                                        | Example |
+| ----------------- | -------------------------------------------------- | -------- |
+| `--target`        | Path of the file or directory to lint              | Lint a single file:<br>`--target=./packages/nodes-base/nodes/Stripe/Stripe.node.ts` <br><br>Lint all files in a directory:<br>`--target=./packages/nodes-base/nodes/Stripe` |
```
[the remark this answers is not in the document]
Comment (ivov) [reply docs/nodes/creating-nodes/nodelinter.md:33]: Same fix for the dir

**Comment by gpene** [comment]
🙏 Thanks @ivov  - updated

**Review by ivov — APPROVED** [review]
🚀

**Pull request merged**